package-lock.json

```diff
@@ -15,47 +15,47 @@
       "resolved": "https://registry.npmjs.org/@babel/code-frame/-/code-frame-7.8.3.tgz",
       "integrity": "sha512-a9gxpmdXtZEInkCSHUJDLHZVBgb1QS0jhss4cPP93EW7s+uC5bikET2twEF3KV+7rDblJcmNvTR7VJejqd2C2g==",
       "requires": {
-        "@babel/highlight": "7.8.3"
+        "@babel/highlight": "^7.8.3"
```

Comment:
I don't think we should change anything on the `package-lock.json`. There are many version changes that could impact the wallet.

Reply:
You are right. All changes on `package-lock.json` were reverted, thanks.